js/components/user/User.js

```diff
@@ -94,7 +95,13 @@ class User extends React.Component {
             return null;
         } else {
             return <Button style={{margin: '0 0.3em 0 0'}} variant='primary' size='sm' ref='submit'
-                           onClick={handlers.onPasswordChange}>{this.i18n('user.password-change')}</Button>;
+                           onClick={isUsingOidcAuth() ?
+                               handlers.onKeycloakRedirect :
+                               handlers.onPasswordChange}>
```

Comment:
I do not like that you combined two different actions into method named _passwordChange():
- change of password when internal auth
- editation of user when external auth

If nothig, the method name is confusing.

Reply:
Resolved with c0fe1368f5cb11269b3f56863ba910e878294afc